### PR TITLE
Write tests mock only certain dependencies, not all.

### PR DESCRIPTION
**Pull request by ArturBorowy**

### CODE REVIEW

File: src/test/java/com/ultimatelogger/multiplatform/ServiceLocatorInitializerTest.kt [hunk 1]
```diff
@@ -8,9 +8,12 @@ import com.ultimatelogger.multiplatform.output.MultiPriorityLogger
 import org.junit.After
 import org.junit.Test
 import org.koin.core.context.stopKoin
+import org.koin.core.module.Module
 
 internal class ServiceLocatorInitializerTest {
 
+    private val mockModule : Module = mock()
+
     private val mockMultiPriorityLogger: MultiPriorityLogger = mock()
```
Comment (KubaB) [inline src/test/java/com/ultimatelogger/multiplatform/ServiceLocatorInitializerTest.kt]: This is no longer used

File: src/test/java/com/ultimatelogger/multiplatform/nonmockist/logggingifisonmultiprioritylogger/TagBuildingTest.kt [hunk 2]
```diff
@@ -0,0 +1,266 @@
...
+                shouldLogThreadName = true
+            )
+        )
+
+        testVMethodCalled(
+            "[main] ($givenFileName:$givenLineNumber) $givenClassName.$givenMethodName()"
+        )
+    }
+
+    private fun testVMethodCalled(expectedTag: String) {
```
Comment (KubaB) [inline src/test/java/com/ultimatelogger/multiplatform/nonmockist/logggingifisonmultiprioritylogger/TagBuildingTest.kt:38]: Maybe move method declaration after the last usage?

**Review by KubaB — CHANGES_REQUESTED** [review]

File: src/test/java/com/ultimatelogger/multiplatform/nonmockist/logggingifisonmultiprioritylogger/TagBuildingTest.kt (hunk 2 above)
Comment (ArturBorowy) [reply src/test/java/com/ultimatelogger/multiplatform/nonmockist/logggingifisonmultiprioritylogger/TagBuildingTest.kt:38]: Rather prefer after first one.

File: src/main/java/com/ultimatelogger/multiplatform/MpUltimateLoggerInitializer.kt [hunk 3]
```diff
@@ -14,19 +16,23 @@ object MpUltimateLoggerInitializer {
...
         setDefaultTagSettings(defaultTagSettings)
 
         ultimateLogger = ultimateLoggerLazy.value
         ultimateLoggerLazy.value.shouldLog = shouldLog
     }
 
-    private fun initServiceLocator(logOutput: MultiPriorityLogger) {
-        ServiceLocatorInitializer.init(logOutput)
+    private fun initServiceLocator(logOutput: MultiPriorityLogger, module: Module) {
+        module.declareDefinition(module.single(override = true) { logOutput  }, Options(override = true))
```
Comment (KubaB) [inline src/main/java/com/ultimatelogger/multiplatform/MpUltimateLoggerInitializer.kt]: Autoformat this line.

**Review by KubaB — CHANGES_REQUESTED** [review]

**Pull request merged**